### DISCUSSION TITLE
Can't send http requests with capactor and axios on android device

### DISCUSSION THREAD

**Discussion by zsjyjishukong** [ionic-team/capacitor#4054] [Q&A]
My server has set access-control-allow-origin: * ,but it still report this error.I built a simple express server and found that the request did not arrive at the server. It should be blocked before sending.
My App :
![WechatIMG51](https://user-images.githubusercontent.com/36651537/104429018-1cc9c880-55c0-11eb-9198-f93bc8bb554a.jpeg)
![WechatIMG50](https://user-images.githubusercontent.com/36651537/104429037-20f5e600-55c0-11eb-9de8-d05bc8c3972c.jpeg)
My express server:
![WX20210113-165623](https://user-images.githubusercontent.com/36651537/104429310-703c1680-55c0-11eb-92bd-e299015631c9.png)

I can get "Hello World!" when I use browser and terminal can console the request infomation, but when I use my capacitor android app, it will report the error, and terminal can't console

**Comment by adonespitogo** [comment]
I'm having the same issue.

**Reply by mattyx96** [reply]
someone resolved guys?

**Reply by adonespitogo** [reply]
If I remember correctly, you have to use the http library from capacitor community to make successful http requests.

**Reply by jeffreysmooths** [reply]
Nice I need something challenging

**Comment by mlynch** [comment]
Hard to see what's going on from the issue here but you shouldn't use `*` for `access-control-allow-origin`, instead you should mirror back the sender origin, if you goal is to have permissive CORS. https://stackoverflow.com/a/19744754
